vite.config.ts

```diff
@@ -46,6 +46,9 @@ export default defineConfig({
 		"import.meta.env.DISPLAY_SLACK_SOCKET_INTEGRATION": process.env.DISPLAY_SLACK_SOCKET_INTEGRATION,
 		"import.meta.env.SENTRY_DSN": JSON.stringify(process.env.SENTRY_DSN),
 		"import.meta.env.TESTS_JWT_AUTH_TOKEN": JSON.stringify(process.env.TESTS_JWT_AUTH_TOKEN),
+		"import.meta.env.ENABLE_MENU_AND_SETTINGS_REDESIGN_FOR_ORGANIZATIONS": JSON.stringify(
```

Comment:
```suggestion
		"import.meta.env.VITE_ENABLE_MENU_AND_SETTINGS_REDESIGN_FOR_ORGANIZATIONS": JSON.stringify(
```

Reply:
renamed to ENABLE_NEW_ORGS_AND_USERS_DESIGN